security/pfSense-pkg-openvpn-client-export/files/usr/local/www/vpn_openvpn_export.php

```diff
@@ -108,6 +108,51 @@
 	$ras_server[] = $ras_serverent;
 }
 
+$a_config =& $config['installedpackages']['openvpnclientexport'];
```

Comment:
This code is looking for stored settings, the page to store the settings was removed but not this code.